extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -254,6 +254,46 @@ public DateTime getStartTime(final String id)
     }
   }
 
+  public Map<String, Object> getMovingAverages(final String id)
+  {
+    log.debug("GetMovingAverages task[%s]", id);
+
+    try {
+      final FullResponseHolder response = submitRequest(
+          id,
+          HttpMethod.GET,
+          "rowStats",
+          null,
+          true
+      );
+      return response.getContent() == null || response.getContent().isEmpty()
+             ? null
+             : jsonMapper.readValue(response.getContent(), new TypeReference<Map<String, Object>>()
+             {
+             });
+    }
+    catch (NoTaskLocationException e) {
+      return null;
+    }
+    catch (IOException e) {
+      throw Throwables.propagate(e);
```

Comment:
can we start moving away form propagates it is going to be removed form guava.

Reply:
Changed this and other `Throwables.propagate` calls in this file